nrpy/c_function.py

```diff
@@ -69,6 +70,7 @@ def __init__(
         ET_current_thorn_CodeParams_used: Optional[List[str]] = None,
         ET_other_thorn_CodeParams_used: Optional[List[str]] = None,
         clang_format_options: str = "-style={BasedOnStyle: LLVM, ColumnLimit: 150}",
+        cfunc_decorators: str = "",
```

Comment:
please place this above cfunc_type here and throughout the Python code to keep the list in order.

Reply:
Also why not just include this in the ctype?

Reply:
The order changes have been made.

Decorators are not types so from an organization and readability perspective, it makes more sense.  Secondly, a single variable (even with a name change) becomes less extensible when considering other programming models and becomes harder to maintain to changes when the number of arguments grow.

Reply:
please place this above cfunc_type here, in the docstring, and in all functions that call it to keep the list in order.

Reply:
this is corrected now.